src/conversions/rust_decimal.rs

```diff
@@ -0,0 +1,216 @@
+#![cfg(feature = "rust_decimal")]
+//! Conversions to and from [rust_decimal](https://docs.rs/rust_decimal)'s [`Decimal`] type.
+//!
+//! This is useful for converting Python's decimal.Decimal into and from a native Rust type.
+//!
+//! # Setup
+//!
+//! To use this feature, add to your **`Cargo.toml`**:
+//!
+//! ```toml
+//! [dependencies]
+//! rust_decimal = "1.0"
+// workaround for `extended_key_value_attributes`: https://github.com/rust-lang/rust/issues/82768#issuecomment-803935643
+#![cfg_attr(docsrs, cfg_attr(docsrs, doc = concat!("pyo3 = { version = \"", env!("CARGO_PKG_VERSION"),  "\", features = [\"rust_decimal\"] }")))]
+#![cfg_attr(
+    not(docsrs),
+    doc = "pyo3 = { version = \"*\", features = [\"rust_decimal\"] }"
```

Comment:
Maybe not encourage `version = "*"` but use `version = ...` to indicate something to replace?

Reply:
I'll add a comment above like:
https://github.com/PyO3/pyo3/blob/ebd417c9ccea884d92792c75fbb14ad833f1eb9a/src/conversions/anyhow.rs#L31
https://github.com/PyO3/pyo3/blob/ebd417c9ccea884d92792c75fbb14ad833f1eb9a/src/conversions/chrono.rs#L21

The others don't have one. I followed num-bigint as an example when writing this.

Reply:
Yep fine to leave like this for now. When we update MSRV above 1.54 we can simplify this and just have the `CARGO_PKG_VERSION` embedded everywhere.

Reply:
I am sorry for re-opening, but I do think @birkenfeld's suggestion of using `version = ...` instead is better. As it stands now, I get the comment about `*` on docs.rs even though docs.rs uses `CARGO_PKG_VERSION` and hence there is no `*` in sight.

Reply:
Sure. I was just trying to match the style of the rest of the code base.

Reply:
The version specification looks good now but it looks like you did not remove the comment about `*`?

Reply:
Took the liberty of amending that commit and will merge this now as I think anything has been addressed now. Any other improvements can be follow-ups.